eclipsePlugin-test/defaultScenario/B.java

```diff
@@ -1,4 +1,4 @@
 public class B {
     // should report URF_UNREAD_FIELD (UrF) and DM_STRING_CTOR (Dm)
-    String s = new String("");
+    String s = "";
```

Comment:
Isn't it a sample bug for unit tests, with a bug purposedely placed here for SpotBugs to find